src/h2olog/h2olog.cc

```diff
@@ -125,7 +136,7 @@ static void show_process(pid_t pid)
     infof("Attaching pid=%d (%s)", pid, cmdline);
 }
 
-static std::string join_str(const std::string &sep, const std::vector<std::string> &strs)
+static const std::string join_str(const std::string &sep, const std::vector<std::string> &strs)
```

Comment:
Returning **const** value is not efficient since C++11 because it disables move semantics. While it is not problem if a function returns const pointer or const reference.

Here is a simple example.

https://gist.github.com/syohex/cfe3cc8fb68b77c4e427fc2230ab5ad6

Reference
- [C++ Move Semantics - The Complete Guide](https://leanpub.com/cppmove) 1.4.1 const Return Values